supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -38,6 +38,26 @@ When it is not possible to remove the terms 'blacklist' and 'whitelist', replace
 * denylist / allowlist
 * blocklist / passlist
 
+=== master and slave
+
+.Example: Removing 'master'
+image:images/no.png[no] A Ceph Monitor maintains a master copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
```

Comment:
Suggest changing "a primary" to "the primary".

Reply:
Also "a master copy" to "the master copy"

Reply:
I also suggest adding "main" as an example to replace the "master" branch in a repo.

Reply:
Added another example for **main**

Reply:
Done. added main example.